# ArgumentError: wrong number of arguments (1 for 2)
#   from (irb):1:in 'calcualte_product'
#   from (irb):4


# The above error means that the method calculate_product() was called without the correct number of arguments. 
# One argument was passed to calculate_product() when the method was expecting two arguments.